client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -64,20 +87,16 @@ const StatsListCard = ( {
 		>
 			{ !! loader && loader }
 			{ !! error && error }
-			<HorizontalBarList data={ data }>
+			<HorizontalBarList>
 				{ data?.map( ( item, index ) => {
-					let rightSideItem;
+					let leftSideItem;
```

Comment:
I didn't convert `leftSideItem` to a function like `rightSideItem` because in the new designs only author avatars and country flags are the right items. It can still be done easily if the requirement changes.